build/getDependencies-windows.sh

```diff
@@ -109,7 +109,7 @@ rm -rf ../src/BloomBrowserUI/bookEdit/test/libsynphony
 #     clean: false
 #     revision: latest.lastSuccessful
 #     paths: {"Palaso.BuildTasks.dll"=>"build/", "*.dll"=>"lib/dotnet"}
-#     VCS: https://github.com/sillsdev/libpalaso.git [NoStrongName]
```

Comment:
These are generated files which use the dependencies that are defined on TC and that the TC builds use. So I guess someone changed something in the "palaso-win32-master Continuous nostrongname (bt436)" configuration. Looking at TC I see that Damien made a change - wasn't he working on adding a nostrongname build configuration to the master branch instead of having a separate NoStrongName branch?